src/tests/networking_tests.rs

```diff
@@ -1518,22 +1520,33 @@ pub mod net_tests {
             );
         });
 
-        assert_eq!(
-            cage.send_syscall(socketpair.sock1, str2cbuf("test"), 4, 0),
-            4
-        );
+        let cage3 = cage.clone();
```

Comment:
I'm curious why we use `clone()` here instead of `fork_syscall()`?

Reply:
We are not really creating a new process here, the `cage` object here is `Arc<Cage>` instead of `Cage`. I need to clone this object so that I can `move` it to the scope of the new thread and use it.